libs/documentation/src/lib/components/formly-multicheckbox/demos/group/multicheckbox-group.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+
+@Component({
+  templateUrl: './multicheckbox-group.component.html',
+
+  selector: `sds-formly-multicheckbox-group-demo`,
+})
+export class MultiCheckboxGroup {
+  form = new FormGroup({});
+  model: any = {};
+  options: FormlyFormOptions = {};
+
+  fields = [
+    {
+      key: 'entity',
+      type: 'multicheckbox',
+      templateOptions: {
+        label: 'Grouped multi-checkbox',
+        groupOptions: {
+          'Contract Opportunities': [
+            {
+              key: 'DE1',
+              value: 'Data entry',
+              tagText: 'Tag',
+            },
+            {
+              key: 'LD',
+              value: 'Listing Data',
+            },
+          ],
+          Entities: [
+            {
+              key: 'POR',
+              value: 'Purpose of Registration',
+            },
+            {
+              key: 'LBN',
+              value: 'Legal Business Name',
+              tooltipText:
```

Comment:
Based on the multi-checkbox template, it looks like there should be a tooltip associated with this option. However in the demo there is none being rendered.